sudoku/easy/script.js

```diff
@@ -0,0 +1,55 @@
+import { addRandom } from '../general/addRandom.js';
+import { checkGrid, checkRowsArr, checkGridArr, checkColumnsArr } from './checkFunctionsRandom.js';
+import { start, pause, reset } from '../general/stopwatch.js';
+
+export let answers = [];
+export let check = document.querySelector('.check');
+export let startGamebtn = document.querySelector('.change');
+let div = document.querySelectorAll('.board>div');
+let matrix = [];
+let arr = [];
+
+const emotyDiv = () => {
+  for (let i = 0; i < div.length; i++) {
+    div[i].innerHTML = '';
+  }
+};
+startGamebtn.addEventListener('click', emotyDiv);
+
+startGamebtn.addEventListener('click', () => {
+  addRandom(div);
+});
+
+function split(arr, len) {
+  let i, n;
+  (i = 0), (n = arr.length);
```

Comment:
Why you wrote like this, instead of just:
```suggestion
  let I = 0;
  let n = arr.length;
```